clockwork_web/templates/settings.html

```diff
@@ -103,28 +103,35 @@ <h1>{{ gettext("User settings %(mila_email_username)s", mila_email_username=mila
 			<div class="col-12">
 
 				<dl class="row">
-					<!--
 					<dt class="col-6">
+						{% if current_user.clockwork_api_key %}
+							<form>
+								<div class="form-group">
+									<label>{{ gettext("CLOCKWORK_API_KEY:") }}</label>
+									<div class="input-group" id="show_hide_password">
+										<input class="form-control" id="clockwork_api_key" type="password" value="{{clockwork_api_key}}">
```

Comment:
The above `if` statement tests for `{{current_user.clockwork_api_key}}`, and yet we then reference `value="{{clockwork_api_key}}"`.

Would that even work? Either `current_user` is redundant or the other one is missing something, I would imagine.